spec/ameba/glob_utils_spec.cr

```diff
@@ -45,6 +45,12 @@ module Ameba
         subject.expand(["**/#{current_file_basename}", "**/#{current_file_basename}"])
           .should eq [current_file_path]
       end
+
+      it "raises an ArgumentError when the glob doesn't match any files" do
+        expect_raises(ArgumentError, "No files found matching foo/*") do
+          subject.expand(["foo/*"])
+        end
+      end
```

Comment:
please make sure to add more test cases:

1. missing files in a glob (already added)
2. missing file
3. missing folder
4. empty folder
5. multiple arguments

Reply:
b2069ea - added extra test cases apart from for empty folder, as per above discussion.